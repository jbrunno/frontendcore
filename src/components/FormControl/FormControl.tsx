import React from 'react';
import {
  FormControl as MuiFormControl,
  FormControlProps as MuiFormControlProps,
} from '@mui/material';

export type FormControlProps = MuiFormControlProps;

export function FormControl(props: FormControlProps) {
  return <MuiFormControl {...props} />;
}

export default FormControl;
