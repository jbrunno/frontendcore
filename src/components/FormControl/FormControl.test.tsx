import React from 'react';
import { render } from '@test-utils/component-testing';
import { FormControl } from './FormControl';

test('renders the FormControl', () => {
  const { container } = render(<FormControl />);
  expect(container).toMatchSnapshot();
});
