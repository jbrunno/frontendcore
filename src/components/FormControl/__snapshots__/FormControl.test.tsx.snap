// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the FormControl 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
  min-width: 0;
  padding: 0;
  margin: 0;
  border: 0;
  vertical-align: top;
}

<div>
  <div
    class="MuiFormControl-root emotion-0"
  />
</div>
`;
